Source/Private/Get-TestConfigs.ps1

```diff
@@ -0,0 +1,18 @@
+function Get-TestConfigs {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory = $true)]
+        $ConfigFiles,
+        [Parameter(Mandatory = $true)]
+        [String]
+        $ConfigsRoot
+    )
+
+    $ConfigFiles.ForEach( {
+            $configName = Remove-Prefix -Value $(Convert-ToUnixPath (Resolve-Path -Path $_ -Relative)) -Prefix './'
+            $configsCmd = -join ($configsCmd, "-c ${ConfigsRoot}/${configName} ")
+        }
+    )
+
+    return $configsCmd
```

Comment:
How would a result with more configs look like? Would it make sense to add an example so that it is easier to see what needs to be handled by the caller?

Reply:
I just moved the existing logic to a separate function because it`s used more than just one time, there are already some tests that covering this. And because this is a private function we don`t expect someone will use it and if someone wants to use this inside this module they can refer to existing examples in Source/Public/Invoke-DockerTests.ps1.